ichnaea/service/geolocate/tests.py

```diff
@@ -57,8 +57,7 @@ def test_ok_cell(self):
 
         self.check_stats(
             counter=[self.metric_url + '.200',
-                     self.metric + '.api_key.test',
-                     self.metric + '.api_log.test.cell_hit']
```

Comment:
The `m.api_log.test.*_hit/miss` metrics are actually the useful ones, those should stay.